src/main/java/io/vertx/spi/cluster/hazelcast/impl/HazelcastAsyncMap.java

```diff
@@ -69,26 +68,26 @@ public HazelcastAsyncMap(VertxInternal vertx, IMap<K, V> map) {
   public Future<Void> put(K k, V v, long ttl) {
     K kk = convertParam(k);
     V vv = convertParam(v);
-    return vertx.executeBlocking(fut -> {
-      map.set(kk, HazelcastServerID.convertServerID(vv), ttl, TimeUnit.MILLISECONDS);
-      fut.complete();
-    }, false);
+    ContextInternal context = vertx.getOrCreateContext();
```

Comment:
why is executeBlocking disappearing here ?

Reply:
Because all `IMap` now implement the HZ async APIs and we no longer need two implementations.